### PR TITLE
TOOLS: ucc_info

### PR DESCRIPTION
**Pull request by Sergei-Lebedev**
## What
ucc_info tool to print information about UCC

## Why ?
Printing various configuration parameters

## How ?
For now only available options is UCC version. Later ucs parser can be used to print configuration parameters of ucc objects such as ucc_lib, ucc_context and ucc_team

### CODE REVIEW

File: tools/info/ucc_info.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (vspetrov) [inline tools/info/ucc_info.c:1]: extra space

File: tools/info/build_info.c [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (vspetrov) [inline tools/info/build_info.c:1]: extra space

File: tools/info/ucc_info.c [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+*/
+
+#include "config.h"
+
+#include "ucc_info.h"
+#include "utils/ucc_parser.h"
+#include <getopt.h>
+#include <stdlib.h>
+
+static void usage() {
```
Comment (vspetrov) [inline tools/info/ucc_info.c:15]: "{" to the newline

**Review by vspetrov — APPROVED** [review]

File: tools/info/ucc_info.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
```
Comment (manjugv) [inline tools/info/ucc_info.h:2]: 2001-2020 ?

**Comment by Sergei-Lebedev** [comment]
@vspetrov fixed

File: tools/info/ucc_info.h (hunk 4 above)
Comment (Sergei-Lebedev) [reply tools/info/ucc_info.h:2]: Yes, I took license header from ucx_info since I reuse ucx_info code structure

File: tools/info/build_info.c [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
```
Comment (manjugv) [inline tools/info/build_info.c:6]: Minor: Indent

File: tools/info/ucc_info.c [hunk 6]
```diff
@@ -0,0 +1,54 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
```
Comment (manjugv) [inline tools/info/ucc_info.c:6]: Minor: Indent

File: tools/info/ucc_info.c [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        }
+    }
+
+    if (print_opts == 0) {
+        usage();
+        return -2;
+    }
+
+
+    if (print_opts & PRINT_VERSION) {
```
Comment (manjugv) [inline tools/info/ucc_info.c:49]: Minor: Extra line.

File: tools/info/ucc_info.h [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+*
```
Comment (manjugv) [inline tools/info/ucc_info.h:3]: Minor: Indent

**Review by manjugv — APPROVED** [review]

**Comment by Sergei-Lebedev** [comment]
@manjugv fixed

**Review by alex--m — APPROVED** [review]

**Pull request merged**